src/fsharp/TypeChecker.fs

```diff
@@ -929,9 +929,15 @@ let TcConst cenv ty m env c =
             | _ -> mkAppTy tcr [TType_measure Measure.One]
         unif measureTy
 
+    let includeExpanded =
```

Comment:
please rename to something more descriptive ... E.g. includeExpandedMeasurables or perhapsexpandedMeasurablesEnabled

Reply:
Makes good sense. I'm on it. :+1: